packages/core/src/elements/Table/interfaces.ts

```diff
@@ -97,6 +99,14 @@ export interface TableColumnProps<T extends object = any> {
 
   minWidth?: string | number;
   maxWidth?: string | number;
+
+  /**
+   * Optional
+   * String options: basic, datetime, alphanumeric. Defaults to alphanumeric.
+   * If a function is passed, it must be memoized. The sortType function should return -1 if rowA is larger, and 1 if rowB is larger. react-table will take care of the rest.
+   * more information about this parameter on the github page https://react-table-omega.vercel.app/docs/api/useSortBy#column-options
+   */
+  sortType?: SortByFn<T> | DefaultSortTypes;
```

Comment:
we are forcing react-table to do manual sorting, please change it to support automatic sort

Reply:
The sorting works correctly without this parameter for strings values. But for the boolean, it works incorrectly.
Try sorting this [example](https://codesandbox.io/s/react-table-example-sort-whtpw?file=/src/App.js) without this parameter by the Status column. In line 120 (of this example) you can uncomment the parameter and try sort again.

P.S. If you know another way to fix the problem sorting by boolean - I'll use your way.